lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>
```

Comment:
OK, I didn't realize this was also for Open edX. How about:

You are not enrolled in any courses yet.

("You're not" or "You aren't" sounds more friendly, but we try to avoid contractions for globalization.)